extensions/integration-tests/src/testConfig.ts

```diff
@@ -66,6 +66,16 @@ var TestingServers: TestServerProfile[] = [
 			database: 'master',
 			provider: ConnectionProvider.SQLServer,
 			version: '2017'
+		}),
+	new TestServerProfile(
+		{
+			serverName: process.env.BACKEND_HOSTNAME + ',' + process.env.BACKEND_PORT,
+			userName:  process.env.BACKEND_USERNAME,
```

Comment:
hmm, we might need the environment variables to be more specific. e.g.: BDC_BACKEND_HOSTNAME, and could you please also make changes to the ADS windows build which already has integration test enabled to include these new environment variables?: https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=28401
You can ask @MattIrv and @kburtram if you need help updating the build definition.

Reply:
btw, the change to the build definition need to be made prior to merging this PR.